build/hydrobuild.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Hydrobuild
+description: Get started with Docker Hydrobuild
+sitemap: false
+---
+
+> **Restricted**
+>
+> Docker Hydrobuild is a early-access product.
+>
+> If you want to get involved in testing Hydrobuild, you can
+> [sign up for the early access program](https://www.docker.com/build-early-access-program/?utm_source=docs).
+{: .restricted }
+
+Hydrobuild is a service that lets you build your container images faster,
+locally and in CI. Builds run on cloud infrastructure optimally dimensioned for
+your workloads, no configuration required. The service uses a remote build
+cache, ensuring fast builds anywhere and for all team members.
+
+## How Hydrobuild works
+
+Using Hydrobuild is no different from running a regular build. You invoke a
+build the same way you normally would, using `docker build`. The difference is
+in where and how that build gets executed.
+
+By default when you invoke a `docker build` command, your build runs on a local
+instance of BuildKit, bundled with the Docker daemon. With Hydrobuild, you send
+the build request to a BuildKit instance running remotely, in the cloud.
+
+The remote builder executes the build steps, and sends the resulting build
+output to the destination that you specify. For example, back to your local
+Docker Engine image store, or to an image registry.
+
+Using Hydrobuild comes with several benefits out of the box:
+
+- Improved build speed
+- Shared build cache
+- Native multi-platform builds
+
+And the best part: you don't need to worry about managing builders or
+infrastructure. Just connect to your builders, and start building.
+
+## Setup
+
+To get started with Hydrobuild, you need to download and install Docker
+Desktop version 4.22.0 or later.
+
+When creating a Hydrobuild builder, you specify a builder endpoint to use. An
+endpoint represents a single, isolated builder. Builder endpoints use the
+following format:
+
+```text
+cloud://<org>/<group>_<platform>
+```
+
+- `<org>` is the Docker organization that the builder is provisioned for
+- `<group>` is the builder node group
+- `<platform>` is the native OS and architecture of the builder
+
+An organization can contain multiple builder groups. Each builder group is an
+isolated builder, and by creating multiple groups you can assign unique
+builders to different teams in the organization.
+
+The platform suffix is optional, and if omitted creates a `linux/amd64` builder
+by default. The supported values for `<platform>` are:
+
+- `linux-amd64`
+- `linux-arm64`
+
+You can use the platform suffix to create a multi-node builder group with
+multiple builders of different native architectures. This gives you a
+high-performance build cluster for building multi-platform images natively. See
+[Create a multi-platform builder](#create-a-multi-platform-builder).
+
+The `cloud://` prefix is optional, and can be omitted.
```

Comment:
It will get added automatically by the CLI. It's mentioned above because it will be visible to the user on `docker buildx ls`

Reply:
The problem is that this isn't currently true for the create command. if you provide the prefix there it will fail with `builder should be in the format: <account>/<builder>`. It expects the non-prefixed format and will prepend `cloud://` itself.
Perhaps we should add support for it for completeness/symmetry?